resources/qml/ToggleButton.qml

```diff
@@ -36,7 +36,7 @@ Switch {
             width: parent.height
             height: width
             radius: width / 2
-            color: toggleButton.down ? "whitesmoke" : "whitesmoke"
+            color: toggleButton.enabled ? "whitesmoke" : "#cccccc"
```

Comment:
This should probably use some palette colors instead of hard-coded values.

Reply:
Yeah, but it didn't before and I just fixed it, because it was obviously broken... stop nitpicking! ;p